builtin/providers/heroku/resource_heroku_app.go

```diff
@@ -9,13 +9,24 @@ import (
 	"github.com/hashicorp/terraform/helper/schema"
 )
 
+type herokuApplication struct {
```

Comment:
Can we document what this is for? 
`application` "is used to store all the details of a heroku app" but now it has a child `App` that is a `herokuApplication` ?